storage/netapp_e/na_eseries_facts.py

```diff
@@ -21,9 +21,21 @@
 module: na_eseries_facts
 version_added: '2.2'
 short_description: Get facts about NetApp E-Series arrays
-extends_documentation_fragment:
-        - auth_basic
 options:
+  api_username:
+    required: true
+    description:
+    - The username to authenticate with the SANtricity WebServices Proxy or embedded REST API.
+  api_password:
+    required: true
+    description:
+    - The password to authenticate with the SANtricity WebServices Proxy or embedded REST API.
+  api_url:
+    required: true
+    description:
+    - The url to the SANtricity WebServices Proxy or embedded REST API.
+    example:
+    - https://prod-1.wahoo.acme.com/devmgr/v2
```

Comment:
There's also a validate_certs parameter that should be added to the documentation (since the basic_auth arg_spec provides it).